src/main/resources/hudson/plugins/git/extensions/impl/CleanBeforeCheckout/config.groovy

```diff
@@ -0,0 +1,7 @@
+package hudson.plugins.git.extensions.impl.CleanBeforeCheckout
+
+def f = namespace(lib.FormTagLib)
+
+f.entry(title: _("Clean submodules"), field: "cleanSubmodules") {
```

Comment:
The question is how the entry should really be named in the UI. I think the help text describes it in a better way. That the git-client-plugin named the parameter `cleanSubmodules` is I think not that correct as cleaning submodules is just a subset of all potentially cleaned up `.git` folders.

Reply:
No objection from me if the option is more descriptive, as in

> Clean submodules and any other subdirectories which contain `.git` directories

Reply:
Renamed option to "Delete untracked nested repositories"